src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -96,23 +130,23 @@ public LocalDateTime getDateInSpecificTimeZone(String dateInString, String zone)
      * OffsetDateTime советуют использовать при записи даты в базу данных.
      */
     public OffsetDateTime offsetDateTime(LocalDateTime localTime) {
-        return OffsetDateTime.now();
+        return OffsetDateTime.of(localTime, ZoneOffset.of(OFFSET_UA));
     }
 
     /**
      * Дана строка в виде "yyyymmdd".
      * Необходимо вернуть Optional даты в LocalDate формате
      */
     public Optional<LocalDate> parseDate(String date) {
-        return Optional.empty();
+        return getOptional(date, YYYY_MM_DD);
```

Comment:
```suggestion
        return getOptional(date, DateTimeFormatter.BASIC_ISO_DATE);
```
you can use existing formatter here